### PR TITLE
fix: bug 1581 - model lastmodified date

### PR DESCRIPTION
**Pull request by pnwbrian**
Now hooks the app reducer and modal actions to set lastmodified to the current system time (in ISO 8601 format)

### CODE REVIEW

File: src/reducers/appsReducer.ts [hunk 1]
```diff
@@ -59,6 +59,7 @@ const appsReducer: Reducer<AppsState> = (state = initialState, action: ActionObj
                 trainingStatus: action.trainingStatus.trainingStatus,
                 // Since we're updating training status simulate update to datetime field
                 datetime: new Date(),
+                lastModifiedDateTime: `${new Date().toISOString().slice(0, 19)}+00:00`,
```
[the remark this answers is not in the document]
Comment (pnwbrian) [reply src/reducers/appsReducer.ts]: created new app reducer block, and expanded action payload to include appid

File: src/reducers/appsReducer.ts [hunk 2]
```diff
@@ -78,7 +78,21 @@ const appsReducer: Reducer<AppsState> = (state = initialState, action: ActionObj
         case AT.EDIT_APP_LIVE_TAG_FULFILLED:
             return { ...state, all: replace(state.all, action.app, app => app.appId) }
         case AT.EDIT_APP_EDITING_TAG_FULFILLED:
-            return { ...state, activeApps: action.activeApps}
+            return { ...state, activeApps: action.activeApps }
+
+        case AT.EDIT_TRAINDIALOG_FULFILLED:
```
Comment (LarsLiden) [inline src/reducers/appsReducer.ts]: Maybe add a TODO here saying need to handle all the other cases?

File: src/reducers/appsReducer.ts [hunk 3]
```diff
@@ -78,7 +78,21 @@ const appsReducer: Reducer<AppsState> = (state = initialState, action: ActionObj
...
+            const app = state.all.find(a => a.appId === action.appId)
+            // User may have delete the app
```
Comment (LarsLiden) [inline src/reducers/appsReducer.ts]: Grammar (copy paste from my exquisite diction)

Comment (pnwbrian) [reply src/reducers/appsReducer.ts]: silly developers ;)

**Pull request merged**